client/state/purchases/test/selectors.js

```diff
@@ -114,6 +116,7 @@ describe( 'selectors', () => {
 				refundAmount: NaN,
 				refundOptions: undefined,
 				refundText: undefined,
+				regularPriceText: undefined,
```

Comment:
Does this PR do anything new about the `regularPriceText` field?

Reply:
No, but the tests were failing without them 🙂

Reply:
For me the tests continue to pass even if I remove this line. If tests were failing for you at some point during development, it was probably for some other reason. Let's remove this change as it's not relevant to the PR.